scripts/run.js

```diff
@@ -0,0 +1,23 @@
+// yarn r [package-short-name] [command]
```

Comment:
What is this doing?

Reply:
I created this script to help isolate commands without going into the folder every time. you can can use `npm run r semantic-ui build` or whatever package you want to target from the root directory